enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+---
+title: audit-command-operation
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-22
+last-updated: 2021-03-02
+status: implementable
+see-also:
+  - [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md)
+  - [custom operator validation #43](https://github.com/operator-framework/enhancements/pull/43)
+  - [Enhancement for validating bundles in a catsrc image #48](https://github.com/operator-framework/enhancements/pull/48)
+  - [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub]
+---
+
+# Audit command operation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions 
+
+1. Should this command to be added to SDK or OPM or in a new tool?
+
+**Context**
+
+The audit command will perform checks that are implemented and are provided via SDK tool commands `bundle validate` and `scorecard` already. Then, it could be done from SDK binary or use it. However, the OPM tool is the tool that has been used with the purpose of admin the catalogs. 
+
+Note that SDK will also get starts to allow users to work with catalogs by using opm. See the proposal [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md).
```

Comment:
While the SDK has some catalog abilities in that EP it was expressed as a [non-goal](https://github.com/operator-framework/enhancements/blob/master/enhancements/operator-sdk-builds-catalogs.md#non-goals)

- Re-implement any part of the catalog management process.
- Add any catalog-related code or files to `operator-sdk` projects, ex. `index.Dockerfile`.
- Add any other functionality other than build a catalog locally (`opm index add`).

Reply:
Your argumentation was added in the open question as well thank you.